src/app/content/practiceQuestions/utils.ts

```diff
@@ -21,23 +22,26 @@ export const getPracticeQuestionsLocationFilters = (
   for (const node of tree) {
     if (isLinkedArchiveTreeSection(node) && summary.countsPerSource[stripIdVersion(node.id)]) {
       const parentId = stripIdVersion(node.parent.id);
-      locationFilters.set(parentId, [...locationFilters.get(parentId) || [], node]);
+      const currentSections = locationFilters.has(parentId)
+        ? assertDefined(locationFilters.get(parentId), 'node has to be there').sections
```

Comment:
i agree that it is silly that typescript can't figure out the `has/get` relationship of maps, but since it can't i've been trying to write code more like

```typescript
const parentFilters = locationFilters.get(parentId);
const currentSections = parentFilters ? parentFilters.sections : [];
```